tools/unsupervised_dataset/SQLCrawler/CrawlerLog.py

```diff
@@ -0,0 +1,76 @@
+import datetime
+import csv
+import os
+import logging
+import pathlib
+
+class CrawlerLog(object):
+    """ Logs the status of the SQL crawler, including websites and queries.
+        
+        The CrawlerLog keeps track of which websites were explored, how many
+        queries were found, and creates a CSV with all the queries. It also
```

Comment:
just a second thought on the output file format (not necessarily to be resolved in the cl:)), if we make it as CSV, later would it be problematic to read the query for later processing? For example, if a multi-line query is not well formatted.